app/frontend/src/components/Cards/Cards.js

```diff
@@ -34,41 +34,4 @@ CopCard.propTypes = {
   size: PropTypes.oneOf(["lg", "sm"]),
 };
 
-function InnerCopCard(props) {
-  return (
-    <div className={combineClasses("card", "inner-cop-card", props.addClass)}>
-      <div className="inner-cop-card-content">{props.children}</div>
-    </div>
-  );
-}
-
-InnerCopCard.propTypes = {
-  addClass: PropTypes.string,
-};
-
-function InnerCopNavCard({ isActive = false, ...props }) {
-  const Tag = "href" in props ? "a" : "button"; // conditionally rendered tagss
-
-  return (
-    <Tag
-      className={combineClasses(
-        "card",
-        "inner-cop-nav-card",
-        isActive ? "active" : undefined,
-        props.addClass
-      )}
-      onClick={props.onClick}
-    >
-      {props.children}
-    </Tag>
-  );
-}
-
-InnerCopNavCard.propTypes = {
-  addClass: PropTypes.string,
-  href: PropTypes.string,
-  isActive: PropTypes.bool,
-  onClick: PropTypes.func,
-};
-
-export { CopCard, InnerCopCard, InnerCopNavCard };
+export { CopCard };
```

Comment:
Let's rename this file after the component

Reply:
@NickTSA This has not been resolved yet. Lets rename this **file** after the component.